build.sbt

```diff
@@ -108,13 +110,42 @@ lazy val server = project
   .in(file("modules/server"))
   .dependsOn(common % "compile->compile;test->test")
   .dependsOn(`internal-core` % "compile->compile;test->test")
-  .dependsOn(channel % "test->test")
-  .dependsOn(monix % "test->test")
-  .dependsOn(fs2 % "test->test")
+  .dependsOn(`internal-monix`% "test->test")
+  .dependsOn(`internal-fs2`% "test->test")
+  .dependsOn(channel)
```

Comment:
Could it be `.dependsOn(channel % "test->test")`?

Reply:
I need it for the `unary` service creation :/